pkg/prowgen/prowgen_config.go

```diff
@@ -115,9 +115,9 @@ func NewGenerateConfigs(ctx context.Context, r Repository, cc CommonConfig, opts
 			copy(options, opts)
 			if isFirstVersion {
 				isFirstVersion = false
-				options = append(options, withNamePromotion(r, branchName))
+				options = append(options, withTagPromotion(r, branchName, true))
 			} else {
-				options = append(options, withTagPromotion(r, branchName))
+				options = append(options, withTagPromotion(r, branchName, false))
```

Comment:
We could just remove the `else { ... }` ?

Reply:
and skip any other generation? because we remove the entire `else` block?

Reply:
no, no skip, it will keep going but only without the `promotion` filed when `!firstVersion`

Reply:
I mean only this else block
```golang
else {
				options = append(options, withTagPromotion(r, branchName))
}
```